docs/blog/20221127-confused-Techie-SunsetMisadventureBackend.md

```diff
@@ -0,0 +1,117 @@
+---
+title: A Sunset and the Misadventures of a Backend
+author: confused-Techie
+date: 2022-11-27
+category:
+  - dev
+  - log
+tag:
+  - backend
+  - sunset
+---
+
+What's the natural response to finding out an application you use is being sunset?
+Spend the next several months creating a complex backend infrastructure of course!
+
+## <!-- more -->
```

Comment:
```suggestion
<!-- more -->
```
The h2 header here makes an `<hr>` and it looks a bit weird formatted? Keep if this was intentional.

Reply:
Super weirdly formatted this was not intentional, must've finished typing it up to late lol. Good catch